x-pack/plugins/endpoint/public/applications/endpoint/types.ts

```diff
@@ -75,17 +75,67 @@ export interface PolicyListState {
 }
 
 /**
- * Policy list store state
+ * Policy details store state
  */
 export interface PolicyDetailsState {
   /** A single policy item  */
-  policyItem: PolicyData | undefined;
+  policyItem?: PolicyData;
   /** data is being retrieved from server */
+  policyConfig?: PolicyDetailsConfig;
   isLoading: boolean;
   /** current location of the application */
   location?: Immutable<EndpointAppLocation>;
 }
 
+export interface PolicyDetailsConfig {
+  windows: WindowsPolicyConfig;
+  mac: MacPolicyConfig;
+  linux: LinuxPolicyConfig;
+}
+
+export interface WindowsPolicyConfig {
```

Comment:
I don't think all of these exports are needed. Please try to only export what you need.